Frontend/src/components/salon-components/salonEmployees.tsx

```
@@ -4,33 +4,32 @@ import { useEffect } from "react";
 import {
-  useBookingActions,
-} from "@/providers/booking-provider";
-import { useEmployeeTechnicianActions, useEmployeeTechnicianState } from "@/providers/employeeTechnician-provider";
+  useEmployeeTechnicianActions,
+  useEmployeeTechnicianState,
+} from "@/providers/employeeTechnician-provider";
 import "@ant-design/v5-patch-for-react-19";
 import { IEmployeeTechnician } from "@/providers/employeeTechnician-provider/context";
 
-const SalonEmployees = ({}: { employeeTechnicians?: IEmployeeTechnician[] }) => {
+const SalonEmployees = ({}: {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)